core/src/main/java/org/kiwix/kiwixmobile/core/error/ErrorActivity.kt

```diff
@@ -113,7 +114,8 @@ open class ErrorActivity : BaseActivity() {
   ${if (allowZims.isChecked) zimFiles() else ""}
   ${if (allowLanguage.isChecked) languageLocale() else ""}
   ${if (allowDeviceDetails.isChecked) deviceDetails() else ""}
-  ${if (allowFileSystemDetails.isChecked) systemDetails() else ""}
+  ${if (allowFileSystemDetails.isChecked) systemDetails() else ""} 
+  ${""}
```

Comment:
literally a blank line, you only need to use `${}` when you are accessing a property or method on a variable

eg:
![image](https://user-images.githubusercontent.com/3358282/96241121-a8b85e00-0f99-11eb-8276-93b926f5439e.png)